cypress/support/components/homepage.js

```diff
@@ -39,10 +40,7 @@ function createNewModel(modelName) {
 
 /** Delete an existent Model */
 function deleteModel(modelName) {
-  cy.server()
-  cy.route('GET', '/apps?**').as('getHomePage')
-  cy.visit('http://localhost:5050')
-  cy.wait('@getHomePage')
+  navigateTo()
```

Comment:
I don't think it's a good practice to nest these functions.  The test should call `navigateTo` and then `deleteModel`